posthog/api/feature_flag.py

```diff
@@ -267,5 +270,10 @@ def my_overrides(self, request: request.Request, **kwargs):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
+@receiver(pre_delete, sender=Experiment)
```

Comment:
Shouldn't this be in the model? :thinking: 

Very hard to find this behavior otherwise.

Reply:
makes sense!